client/core/trade.go

```diff
@@ -476,11 +482,13 @@ func (t *trackedTrade) isRedeemable(match *matchTracker) bool {
 			return false
 		}
 		assetCfg := t.wallets.toAsset
+		log.Debugf("coin confs = %d, swapConf = %d", confs, assetCfg.SwapConf)
 		return confs >= assetCfg.SwapConf
 	}
 	if dbMatch.Side == order.Taker && metaData.Status == order.MakerRedeemed {
 		return true
 	}
+	log.Debugf("match side = %v, status = %v", dbMatch.Side, metaData.Status)
```

Comment:
This is kind of loud when `status = MatchComplete`

Reply:
Agreed.  These should be remove from the tracked trades.  It's part of why I'm always pushing for more logging, because otherwise things like this can be neglected.